test/travis-ci-run.sh

```diff
@@ -28,7 +28,10 @@ done
 curl --silent "http://localhost:8080/" | grep --silent "meta.*Generator.*Thunder"
 
 # Check that relevant modules are installed
-docker exec "${TEST_THUNDER_PHP_DOCKER_ID}" su - thunder --command='cd www; drush pm:list --status=enabled --format=json;' | jq --exit-status '.thunder_performance_measurement, .testsite_builder, .media_entity_generic'
+docker exec "${TEST_THUNDER_PHP_DOCKER_ID}" su - thunder --command='cd /home/thunder/www/docroot; drush pm:list --status=enabled --format=json;' | jq --exit-status '.thunder_performance_measurement, .media_entity_generic'
+
+# Workaround for testsite_builder until Drush is fixed
```

Comment:
Please add @todo with the drush issue here, so we can remove the workaround later